From 816e9c33582044f2af79d64ff25a2d86439a7eac Mon Sep 17 00:00:00 2001
From: Liang Chen <cl@rock-chips.com>
Date: Thu, 8 Feb 2018 21:00:13 +0800
Subject: [PATCH] arm: dts: move memory of ramoops to 46M for rk312x/rk322x

We put memory of ramoops on 46M from now on for arm32, beacause some
board only have 128M memory.

Change-Id: Ie94a7619bfe006e196d3269711a499acbd7c87b9
Signed-off-by: Liang Chen <cl@rock-chips.com>
---
 arch/arm/boot/dts/rk312x-android.dtsi | 4 ++--
 arch/arm/boot/dts/rk322x-android.dtsi | 4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm/boot/dts/rk312x-android.dtsi b/arch/arm/boot/dts/rk312x-android.dtsi
index e4ed16e0df0f..b9e4bbdcd80e 100644
--- a/arch/arm/boot/dts/rk312x-android.dtsi
+++ b/arch/arm/boot/dts/rk312x-android.dtsi
@@ -129,8 +129,8 @@
 			reg = <0x88000000 0x1800000>;
 		};
 
-		ramoops_mem: ramoops@68000000 {
-			reg = <0x68000000 0xf0000>;
+		ramoops_mem: ramoops@62e00000 {
+			reg = <0x62e00000 0xf0000>;
 		};
 
 		drm_logo: drm-logo@00000000 {
diff --git a/arch/arm/boot/dts/rk322x-android.dtsi b/arch/arm/boot/dts/rk322x-android.dtsi
index f19ae5668ca7..29b0f444663f 100644
--- a/arch/arm/boot/dts/rk322x-android.dtsi
+++ b/arch/arm/boot/dts/rk322x-android.dtsi
@@ -92,8 +92,8 @@
 			reg = <0x0 0x0>;
 		};
 
-		ramoops_mem: ramoops@68000000 {
-			reg = <0x68000000 0xf0000>;
+		ramoops_mem: ramoops@62e00000 {
+			reg = <0x62e00000 0xf0000>;
 		};
 	};
 
-- 
2.35.3

